tdrs-backend/tdpservice/search_indexes/management/commands/clean_and_reparse.py

```diff
@@ -38,22 +38,22 @@ def __get_log_context(self, system_user):
         """Return logger context."""
         context = {'user_id': system_user.id,
                    'action_flag': ADDITION,
-                   'object_repr': "Clean and Re-parse"
+                   'object_repr': "Clean and Reparse"
                    }
         return context
 
     def __backup(self, backup_file_name, log_context):
         """Execute Postgres DB backup."""
         try:
-            logger.info("Beginning re-parse DB Backup.")
+            logger.info("Beginning reparse DB Backup.")
             call_command('backup_db', '-b', '-f', f'{backup_file_name}')
-            logger.info("Backup complete! Commencing clean and re-parse.")
+            logger.info("Backup complete! Commencing clean and reparse.")
 
             log("Database backup complete.",
                 logger_context=log_context,
                 level='info')
         except Exception as e:
-            log("Database backup FAILED. Clean and re-parse NOT executed. Database and Elastic are CONSISTENT!",
+            log("Database backup FAILED. Clean and reparse NOT executed. Database and Elastic are CONSISTENT!",
                 logger_context=log_context,
                 level='error')
             raise e
```

Comment:
in the below function call of `__handle_elastic`, we should always set delete and new indices to happen given my above comment.